src/statsig/constants.ts

```diff
@@ -115,4 +117,10 @@ export const DynamicConfigs = {
       rewardReminderDate: new Date(0).toISOString(),
     },
   },
+  [StatsigDynamicConfigs.POINTS_CONFIG]: {
+    configName: StatsigDynamicConfigs.POINTS_CONFIG,
+    defaultValues: {
+      pointsMetadata: [] as PointsMetadata[],
```

Comment:
The type here may seem a bit wonky at first, but I wanted something that checks the following boxes:
* Can have new pieces of metadata added while maintaining backwards compatibility
* Supports explicit card ordering

I toyed around with a few other ideas before settling on this one, but happy to consider alternatives!

Reply:
i think this config shape is fine and i like the idea of supporting explicit card ordering, but the designs / wallet logic don't support this which is a shame 😅